pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -494,6 +484,26 @@ func (plugin *kubenetNetworkPlugin) Status() error {
 	return nil
 }
 
+// getContainerID returns the containerID that will be used to retrieve the pod IP.
+// For Docker, the container ID is the pod infra container's ID.
+// For rkt, the container ID is the API pod's UID.
+// TODO(yifan): Rename to getPodID.
```

Comment:
We probably will not end up with getPodID. It is ambiguous. There are too many IDs associated with a pod.